app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+from app.errors import NotVaccinatedError,\
```

Comment:
Import each error from `app.errors` on the new line with the use of parenthesis.